Frends.AzureBlobStorage.CreateContainer/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.0.1] - 2022-11-28
```

Comment:
This is new functionality, so minor version should be updated instead of build version.